build_fenics_deps.sh

```diff
@@ -1,26 +1,27 @@
 #!/bin/bash
 
-if [ ${CONDA_BUILD_TYPE} = host-gcc ]; then
+if [ "${CONDA_BUILD_TYPE}" = "host-gcc" ]; then
     # Dependencies
-#     conda build subprocess32/host-gcc
-#     conda build eigen3/host-gcc
-#     conda build boost/host-gcc
-#     conda build hdf5-parallel/host-gcc
-#     conda build h5py-parallel/host-gcc
-#     conda build vtk/host-gcc
+    conda build subprocess32/host-gcc
+    conda build eigen3/host-gcc
+    conda build boost/host-gcc
+    conda build hdf5-parallel/host-gcc
+    conda build h5py-parallel/host-gcc
+    conda build vtk/host-gcc
     conda build petsc/host-gcc
-#     conda build petsc4py/host-gcc
-#     conda build slepc/host-gcc
+    conda build petsc4py/host-gcc --no-remove-work-dir
```

Comment:
Eg fekk opp ei feilmelding om at "work dir" som skulle brukast i testing var sletta (ein feature i nye conda build). Eg fann fort ut at to mulige løysingar var (1) å endre meta.yaml, (2) å legge til --no-remove-work-dir. Sistnevnte var "the path of least resistance", men det er nok meir korrekt å endre meta.yaml. Vi kan evt. endre meta.yaml no når vi har fått det til å fungere ein gong og prøve igjen uten --no-remove-work-dir.